rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -113,78 +113,60 @@ DB_PASSWORD=${DB_PASSWORD}
 EOL
 
 # Config : Directories
-  cat > ${LIBRENMS_PATH}/config.d/directories.php <<EOL
-<?php
-\$config['install_dir'] = '${LIBRENMS_PATH}';
-\$config['log_dir'] = '/data/logs';
-\$config['rrd_dir'] = '/data/rrd';
+cat > ${LIBRENMS_PATH}/database/seeders/config/directories.yaml <<EOL
+install_dir: '${LIBRENMS_PATH}'
+log_dir: /data/logs
+rrd_dir: /data/rrd
 EOL
 ln -sf /data/logs ${LIBRENMS_PATH}/logs
 
 # Config : Server
-  cat > ${LIBRENMS_PATH}/config.d/server.php <<EOL
-<?php
-\$config['own_hostname'] = '$(hostname)';
-\$config['base_url'] = '${LIBRENMS_BASE_URL}';
+cat > ${LIBRENMS_PATH}/database/seeders/config/server.yaml <<EOL
+own_hostname: '$(hostname)'
+base_url: '${LIBRENMS_BASE_URL}'
 EOL
 
 # Config : User
-cat > ${LIBRENMS_PATH}/config.d/user.php <<EOL
-<?php
-\$config['user'] = "librenms";
-\$config['group'] = "librenms";
+cat > ${LIBRENMS_PATH}/database/seeders/config/user.yaml <<EOL
+user: librenms
+group: librenms
 EOL
 
 # Config : Fping
-cat > ${LIBRENMS_PATH}/config.d/fping.php <<EOL
-<?php
-\$config['fping'] = "/usr/sbin/fping";
-\$config['fping6'] = "/usr/sbin/fping6";
+cat > ${LIBRENMS_PATH}/database/seeders/config/fping.yaml <<EOL
+fping: /usr/sbin/fping
+fping6: /usr/sbin/fping6
 EOL
 
 # Config : ipmitool
-cat > ${LIBRENMS_PATH}/config.d/ipmitool.php <<EOL
-<?php
-\$config['ipmitool'] = "/usr/sbin/ipmitool";
+cat > ${LIBRENMS_PATH}/database/seeders/config/ipmitool.yaml <<EOL
+ipmitool: /usr/sbin/ipmitool
 EOL
 
-# Config : Disable autoupdate
+# Config : Disable autoupdate (set in config.php so it cannot be overridden in the webui)
 cat > ${LIBRENMS_PATH}/config.d/autoupdate.php <<EOL
 <?php
 \$config['update'] = 0;
 EOL
 
 # Config : Services
-cat > ${LIBRENMS_PATH}/config.d/services.php <<EOL
-<?php
-\$config['show_services'] = 1;
-\$config['nagios_plugins'] = "/usr/lib/monitoring-plugins";
+cat > ${LIBRENMS_PATH}/database/seeders/config/services.yaml <<EOL
+show_services: true
+nagios_plugins: /usr/lib/monitoring-plugins
 EOL
 
-# Config : Memcached
-if [ -n "${MEMCACHED_HOST}" ]; then
-    cat > ${LIBRENMS_PATH}/config.d/memcached.php <<EOL
-<?php
-\$config['memcached']['enable'] = true;
-\$config['memcached']['host'] = '${MEMCACHED_HOST}';
-\$config['memcached']['port'] = ${MEMCACHED_PORT};
-EOL
-fi
```

Comment:
Any reason to remove the memcached config?

Reply:
These settings don't exist in LibreNMS.